### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by spyderdsn** [MagicSketch/MagicMirror#92]
I think this project is abandoned now in favour of https://angle.sh/ 

“MagicMirror3 2.sketchplugin” is damaged and can’t be opened. You should move it to the Bin.

Sketch Version 61 (89581)

**Comment by blakeperdue** [comment]
Yes, I would say it's abandoned. It's a shame since they sold this tool to people then provided no future support for it as Sketch evolved and their plugin broke. For those of you who paid for this tool, I'd seek a refund or some sort of compensation for their lack of commitment in a product they charged for.

**Comment by ivancucer** [comment]
Sadly the author is dead. Rest in peace @jamztang

**Comment by DasBlatz** [comment]
> Sadly the author is dead. Rest in peace @jamztang

Seriously?

**Comment by otri** [comment]
As you can see from @jamztang commit activity, he's clearly alive and cranking on code.  It's too bad this project has gone dark, because Angle has expanded rapidly into a billion things I don't want or need, and overly commercialized.   Would be nice to resurrect this plugin or integrate its functionality into [https://github.com/sureskumar/sketch-isometric](https://github.com/sureskumar/sketch-isometric)
